content/_changelogs/9.5.3.md

```diff
@@ -0,0 +1,43 @@
+## 9.5.3
+
+_Released 3/28/2022_
+
+**Bugfixes:**
+
+- Corrected an logging issue with `cy.intercept()` where the response status
```

Comment:
This should be `a` not `an`

Reply:
fixed: [`563831a` (#4407)](https://github.com/cypress-io/cypress-documentation/pull/4407/commits/563831a060754204c8902bf6880124f85823e3f7)